stats/projects_Xed_graphs.php

```diff
@@ -4,24 +4,36 @@
 include_once($relPath.'project_states.inc');
 include_once($relPath.'theme.inc');
 include_once($relPath.'misc.inc'); // get_enumerated_param()
+include_once($relPath.'graph_data.inc');
 
 require_login();
 
 $which = get_enumerated_param($_GET, 'which', null, $project_status_descriptors);
 
 $psd = get_project_status_descriptor($which);
 
-output_header($psd->graphs_title);
+$charts = [
+    ["id" => "curr_month_proj", "config" => curr_month_proj($which)],
+    ["id" => "cumulative_month_proj", "config" => cumulative_month_proj($which)],
+    ["id" => "total_proj_graph", "config" => total_proj_graph($which)],
+    ["id" => "cumulative_total_proj_graph", "config" => cumulative_total_proj_graph($which)],
+];
 
-echo "<h1>$psd->graphs_title</h1>";
+$js_data = '$(function(){';
```

Comment:
Not for this MR, but perhaps we need a function to create this JS and reuse it in these various files?

Reply:
Yeah, once we have a clear pattern after going through all the graphs, this is something we should definitely look into.